### DISCUSSION TITLE
Flowdeck 'thetapix' parameter

### DISCUSSION THREAD

**Discussion by Ooip83** [bitcraze/discussions#1721] [Q&A]
Hi! In the 'mm_flow.c' code, why is the calculation of 'thetapix' parameter using sine instead of tangent? I found that sine is used when spherical projection model is used. Is it the spherical projection model valid for all PMW3901 sensor? As I am going to implement Flowdeck V2 in Arduino.

**Comment by gemenerik** [comment]
It looks like the sine is used to calculate the 'chord length'. This should work for other PMW3901 sensors, but I can't guarantee it universally since we’ve only tested it with our setup.
